src/main/java/rick/Ui.java

```diff
@@ -0,0 +1,96 @@
+package rick;
+
+/**
+ * Main Formatter class to format the UI for the rick.Rick chatbot in the terminal.
+ *
+ * @author SeeuSim
+ *         AY2223-S2 CS2103T
+ */
+public class Ui {
```

Comment:
I like the way you implement your Ui methods helps to provide the output, it is very neat